src/solution/react/Component.js

```diff
@@ -25,6 +25,6 @@ class Component {
     }
 }
 
-Component.prototype.isReactComponent = {};
+Component.prototype.isReactComponent = true;
```

Comment:
Må endre test for oppgave 11 for dette. Fra `toEqual({})` til `toBe(true)`.

Reply:
Jeg skal gå gjennom resten etterhvert og se hva som må oppdateres for å passe den virtuelle-dom'en.